.github/CONTRIBUTING.md

```diff
@@ -118,45 +119,83 @@ This project uses [Prettier](http://prettier.io/) for code formatting. Consider
 
 ## Commiting
 
-- Never use the `-m <msg>` / `--message=<msg>` flag to `git commit`;
+A commit message can consists of a **header**, **body** and **footer**. The header is the only mandatory part and consists of a type and a subject. The body is used to fully describe the change. The footer is the place to reference any issues or pull requests related to the commit. That said, we end with a template like this:
+
+```
+<type>: <subject>
+
+[optional body]
+
+[optional footer]
+```
+
+To ensure that a commit is valid, easy to read, and changelog-ready, we have a hook that lints the commit message before allowing a commit to pass. This linter verifies the following:
+
+- The header (first line) is the only mandatory part of the commit message.
+- The header should contains:
+  - A type:
+    - Must be lowercase;
+    - Must be one of:
+      - **chore**: A change that neither fix a bug or adds a feature;
```

Comment:
`...neither fix a bug or...` -> `...neither fixes a bug nor...`

`neither` should be paired with `nor` instead of `or`